lg_earth/src/lg_earth/kmlsync.py

```diff
@@ -330,10 +330,24 @@ class KmlQueryHandler(tornado.web.RequestHandler):
     def initialize(self):
         self.playtour = self.application.playtour
         self.playtour_service = self.application.playtour_service
+        self.planet = self.application.planet
+        self.planet_service = self.application.planet_service
+        self.get_planet = self.application.get_planet
+
+    def _wait_for_planet(self, planet):
```

Comment:
> there's absolutely no guarantee that when `_wait_for_planet` succesfully
> finishes then the planet is really changed. We're observing kmlsync http
> server that's polled by GE from time to time.

Well, there's the guarantee that either it's reporting the new planet on the
viewsync stream, or that five seconds have passed and it gave up waiting. Five
seconds is, of course, arbitrarily chosen, but at least on my box it appears
not to broadcast the planet on the viewsync packets until it has changed to
that planet.